### DISCUSSION TITLE
Scene preview in indoor scenario

### DISCUSSION THREAD

**Discussion by fsvsf** [NVlabs/sionna#264] [Q&A] [answered]
Hello

I am currently loading an indoor scene (a 3D big factory hall model) and using ray tracing to see some channel parameters. My problem is that the roof of the model makes it difficult to see what is actually happening in the indoor scenario. I need to move around with the mouse to find the place where my transmitter and receiver are located. 

I really would love to have a way of having the roof with its properties but not seeing it on the preview scene. I tried to solve it by changing the .xml file exported in blender and put the material of the roof as a "null" bsdf material. However, when I call the preview() function I still see a roof with a random color being placed. 

Is there a way to put an item invisible so I can see more clearly my scene? 

Thank you in advance for your response.

**Comment by jhoydis** [comment, marked as answer]
Hi,

That's a problem that we are aware of and plan to look into.
Your approach of setting the material to a "null" BSDF is a good idea which could probably work when you call ``scene.render`` and not ``scene.preview``.

**Reply by fsvsf** [reply]
Hi,

it worked with scene.render and now I am able to see better my scenario. The only drawback is that I cannot move the camera freely as before.

Thank you for your response

**Reply by merlinND** [reply]
Hi @fsvsf,

The next Sionna release will add a new argument `clip_at` to the `preview()` function that allows cutting out any part of the scene, e.g. to peek inside of buildings.

**Reply by jasperbrg** [reply]
Hi everyone,

I have a related request for rendering the transmission paths. It would be very helpful for rendering small indoor scenarios if the width of the cylinders representing the paths was adjustable, such that many paths in a small environment can be distinguished.

Also, a possibility to remove a scene object from the rendering completely or to assign an alpha value to it would be great for these scenarios. As far as I can see, the Mitsuba format does not natively include alpha values but please correct me if I'm wrong.

Thanks!
